### PR TITLE
fix: handle out_of_gas_error on aex141 cleanup

### PR DESCRIPTION
**Pull request by jyeshe**
Avoid deleting aex141 contract indexed by metadata covering some mainnet cases.

### CODE REVIEW

**Comment by jyeshe** [comment]
Thanks for finding it @sborrazas

File: lib/ae_mdw_web/helpers/aexn_helper.ex [hunk 1]
```diff
@@ -25,7 +25,9 @@ defmodule AeMdwWeb.Helpers.AexnHelper do
   @spec enc_id(pubkey()) :: String.t()
   def enc_id(pk), do: :aeser_api_encoder.encode(:account_pubkey, pk)
 
-  @spec sort_field_truncate(String.t()) :: String.t()
+  @spec sort_field_truncate(String.t() | :atom) :: String.t()
```
Comment (sborrazas) [inline lib/ae_mdw_web/helpers/aexn_helper.ex:28]: this should be `atom()` since `:atom` represents that literal atom

rest looks good 👍

Comment (jyeshe) [reply lib/ae_mdw_web/helpers/aexn_helper.ex:28]: true, atom atom (:

**Review by thepiwo — APPROVED** [review]

**Comment by jyeshe** [comment]
rebased with new Dockerfile

**Review by sborrazas — APPROVED** [review]

**Pull request merged**